drivers/exec/driver.go

```diff
@@ -192,6 +192,12 @@ func (d *Driver) buildFingerprint() *drivers.Fingerprint {
 		HealthDescription: "healthy",
 	}
 
+	if !utils.IsUnixRoot() {
```

Comment:
Moved this up to avoid logging cgroup errors and the like when we don't have permissions